src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -730,8 +730,7 @@ ImmutableArray<ICustomModifier> RefCustomModifiers
         {
             get;
         }
-        
-        //PROTOTYPE(readonlyRefs): need to add another API to distinguish readonly, for now treat them the same
```

Comment:
Was an issue opened on this?

Reply:
[https://github.com/dotnet/roslyn/issues/21976](https://github.com/dotnet/roslyn/issues/21976)

---
In reply to: [138729016](https://github.com/dotnet/roslyn/pull/21982#discussion_r138729016) [](ancestors = 138729016)